x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/utils.tsx

```diff
@@ -0,0 +1,318 @@
+        values: {
+          syncs: inProgressCount,
+        },
+      });
+export const getRunningJobsTooltip = (inProgressCount: number, isCrawler: boolean) => (
+  <EuiText size="xs">
+    {!isCrawler ? (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.connectorStats.runningTooltip"
+        defaultMessage="{inProgressCount} {runningCountText} - Number of running sync jobs (This includes Idle Syncs)"
+        values={{
+          inProgressCount: <b>{inProgressCount}</b>,
+          runningCountText: (
+            <b>
+              {i18n.translate('xpack.enterpriseSearch.connectorStats.connectorTooltipRunning', {
+                defaultMessage: `Running syncs`,
+              })}
+            </b>
+          ),
+        }}
+      />
+    ) : (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.connectorStats.runningCrawlerTooltip"
+        defaultMessage="{inProgressCount} {runningCountText} - Number of running crawls"
+        values={{
+          inProgressCount: <b>{inProgressCount}</b>,
+          runningCountText: (
+            <b>
+              {i18n.translate('xpack.enterpriseSearch.connectorStats.crawlerTooltipRunning', {
+                defaultMessage: `Running crawls`,
+              })}
+            </b>
+          ),
+        }}
+      />
+    )}
+  </EuiText>
+);
+export const getRunningJobsBadgeAriaLabel = (inProgressCount: number, isCrawler: boolean) =>
+  !isCrawler
+    ? i18n.translate('xpack.enterpriseSearch.connectorStats.runningBadgeAriaLabel', {
+        defaultMessage: '{number} Running syncs.',
+        values: {
+          number: inProgressCount,
+        },
+      })
+    : i18n.translate('xpack.enterpriseSearch.connectorStats.runningCrawlerBadgeAriaLabel', {
+        defaultMessage: '{number} Running crawls.',
+        values: {
+          number: inProgressCount,
+        },
+      });
+export const getRunningJobsBadgeLabel = (inProgressCount: number, isCrawler: boolean) =>
+  !isCrawler
+    ? i18n.translate('xpack.enterpriseSearch.connectorStats.runningBadgeLabel', {
+        defaultMessage: '{number} Running syncs',
+        values: {
+          number: inProgressCount,
+        },
+      })
+    : i18n.translate('xpack.enterpriseSearch.connectorStats.runningCrawlerBadgeLabel', {
+        defaultMessage: '{number} Running crawls',
+        values: {
+          number: inProgressCount,
+        },
+      });
+export const getIdleJobsLabel = (idleCount: number) =>
+  i18n.translate('xpack.enterpriseSearch.connectorStats.idleJobsText', {
+    defaultMessage: '{idleCount} Idle syncs',
+    values: {
+      idleCount,
+    },
+  });
+export const getIdleJobsTooltip = (idleCount: number) => (
+  <EuiText size="xs">
+    <FormattedMessage
+      id="xpack.enterpriseSearch.connectorStats.idleTooltip"
+      defaultMessage="{idleCount} {idleCountText} - Number of sync jobs which are still running but have not seen any update from the backend connector for more than 1 minute"
```

Comment:
```suggestion
      defaultMessage="{idleCount} {idleCountText} - Number of sync jobs which are still running, but have not been updated by the backend connector for more than 1 minute."
```

@danajuratoni I'm not sure "backend connector" is clear to users here, what's a clearer term